### ISSUE TITLE
HELPPPPPPPPPPPPPPPPP

### ISSUE THREAD

**Issue by Harry007fv** [NightfallGT/Discord-Account-Generator#64]
it's really complicated to just make a tutorial video

when I go to the cmd that I try to launch python it sends me back to the microsoft page except that it is version 9.1 but I install it anyway then I try to install the extention but the chrome and the request extention are not possible to be installed because I have a pip version 21.2 instead of 21.3

tell me everything I need to do from the start, because the tutorial provided is sucks

**Comment by epyklion** [comment]
Get python from the https://www.python.org/ instead of the microsoft store. Make sure to select add to PATH during the installation.

**Comment by arin2115** [comment]
No, that's your problem if you can't even install python properly
